src/ui/public/error_auto_create_index/error_auto_create_index.html

```diff
@@ -1,23 +1,35 @@
 <div class="app-container kbnError--auto-create-index euiText">
-  <h3><icon aria-hidden="true" type="'alert'" color="'danger'"></icon> Oh no!</h3>
+  <h3><icon aria-hidden="true" type="'alert'" color="'danger'"></icon>
+    <span i18n-id="common.ui.errorAutoCreateIndex.errorTitle"
+    i18n-default-message="Oh no!"></span>
+  </h3>
 
-  <p>
-    It looks like your Elasticsearch cluster's <code>action.auto_create_index</code> setting is
+  <p i18n-id="common.ui.errorAutoCreateIndex.errorDescription"
+    i18n-default-message="It looks like your Elasticsearch cluster's {autoCreateIndexActionConfig} setting is
     preventing Kibana from automatically creating an index to store saved objects. Kibana uses this
     Elasticsearch feature because it is the best way to make sure that the saved object index uses
-    the correct mappings/schema, and it allows Kibana to poll Elasticsearch less often.
-  </p>
+    the correct mappings/schema, and it allows Kibana to poll Elasticsearch less often."
+    i18n-values="{ html_autoCreateIndexActionConfig: '<code>action.auto_create_index</code>' }"
+  ></p>
 
-  <p>Unfortunately, until this issue is fixed you won't be able to save anything in Kibana.</p>
-
-  <h3>Ok, how do I fix this?</h3>
+  <p i18n-id="common.ui.errorAutoCreateIndex.errorDisclaimer"
+    i18n-default-message="Unfortunately, until this issue is fixed you won't be able to save anything in Kibana."
+  ></p>
 
+  <h3 i18n-id="common.ui.errorAutoCreateIndex.fixErrorTitle"
+    i18n-default-message="Ok, how do I fix this?"
+  ></h3>
   <ol>
-    <li>
-      Remove <code>action.auto_create_index: false</code> from your Elasticsearch configuration file
-    </li>
-    <li>Restart elasticsearch.</li>
-    <li>Use the browser's back button to return to what you were doing.</li>
+    <li i18n-id="common.ui.errorAutoCreateIndex.fixErrorListItemOne"
```

Comment:
Please do not use ItemOne, ItemTwo in id names. Instead add more descriptive id which describes message it is used for